pkg/apiserver/middlewares/audit/audit.go

```diff
@@ -340,40 +340,31 @@ func (r responseBodyWriter) WriteString(s string) (n int, err error) {
 }
 
 func getPostObjectName(c *gin.Context) string {
-	// get request body
-	data, err := c.GetRawData()
-	if err != nil {
-		clog.Warn("[audit] get raw data err: %s", err.Error())
-		return ""
-	}
-	// put request body back
-	c.Request.Body = ioutil.NopCloser(bytes.NewBuffer(data))
+	obj := struct {
+		Metadata struct {
+			Name string `json:"name"`
+		} `json:"metadata"`
+	}{}
+
 	// get resource name from metadata.name
```

Comment:
Please modify the meaning of this comment to express the meaning of the function.